packaging/linux/buildpackage.sh

```diff
@@ -55,8 +55,14 @@ popd > /dev/null
 
 # Add native libraries
 echo "Downloading dependencies"
-curl -s -L -O https://github.com/OpenRA/AppImageSupport/releases/download/${DEPENDENCIES_TAG}/libs.tar.bz2 || exit 3
-curl -s -L -O https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage || exit 3
+if $(command -v curl >/dev/null 2>&1); then
```

Comment:
`   ^-- SC2091: Remove surrounding $() to avoid executing output.`

I guess you missed my comment above while dealing with @Phrohdoh's.

As you've already checked that the user has curl or wget this pattern can simplify further to

```
echo "Downloading dependencies"
if command -v curl >/dev/null 2>&1; then
    curl -s -L -O https://github.com/OpenRA/AppImageSupport/releases/download/${DEPENDENCIES_TAG}/libs.tar.bz2 || exit 3
    curl -s -L -O https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage || exit 3
else
    wget -cq https://github.com/OpenRA/AppImageSupport/releases/download/${DEPENDENCIES_TAG}/libs.tar.bz2 || exit 3
    wget -cq https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage || exit 3
fi
```

Reply:
Ah, sorry, 'tis done now.